### PR TITLE
Better doc

### PR DESCRIPTION
**Pull request by FabienLelaquais**
Clarify get_name()/get_js_module_name() a little bit.

### CODE REVIEW

File: taipy_gui_ext_library/frontend/webpack.config.js [hunk 1]
```diff
@@ -23,11 +23,18 @@ module.exports = (env, options) => {
     mode: options.mode, // 'development' | 'production'
     entry: ["./src/index.ts"],
     output: {
+      // filename: the filename of the JavaScript bundle that gets generated.
+      // The 'get_scripts()' method of the ElementLibrary subclass must return
+      // the full path to this file.
       filename: "library.js",
       path: path.resolve(__dirname, "dist"),
       library: {
+        // name: must a Camel-case version of the extension library name
```
Comment (FredLL-Avaiga) [inline taipy_gui_ext_library/frontend/webpack.config.js]: must *be* a Camel...

Comment (FredLL-Avaiga) [reply taipy_gui_ext_library/frontend/webpack.config.js]: (the default implementation of get_js_name())

**Review by FredLL-Avaiga — APPROVED** [review]
👍

File: taipy_gui_ext_library/frontend/webpack.config.js (hunk 1 above)
Comment (FabienLelaquais) [reply taipy_gui_ext_library/frontend/webpack.config.js]: As long as your `get_js_name()` really means `get_js_module_name()`, I'm good with that.

**Pull request merged**